### ISSUE TITLE
Consider using a shebang `/bin/bash` instead of `/bin/sh` for `configure`

### ISSUE THREAD

**Issue by Silex** [NixOS/nix#4896]
Hello,

I build from source inside an alpine docker container and no matter what I tried I always got `./configure: line 7906: syntax error: unexpected "(" (expecting "fi")`

I tried `bash -c './configure'`, using the Dockerfile `SHELL` command to switch to bash... but the only thing that worked was changing the shebang for `/bin/bash` instead of `/bin/sh`.

**Comment by Ericson2314** [comment]
Makes sense to me. If we are using bashism we should either require bash or remove the bashisms.

**Comment by sternenseemann** [comment]
`/usr/bin/env bash` would be more sensible, I suspect we are using `/bin/sh` in the first place because it is essentially the `/bin/bash` of NixOS.

**Comment by grahamc** [comment]
We should remove the bashisms. /bin/sh is posix, /bin/bash isn't. Nix's sandbox doesn't include /usr/bin/env, either.

**Comment by xokdvium** [comment]
This is no longer relevant, since the autotools build system has been replaced with meson.
